--- node_modules/yun-elp/es/src/table/src/table-script-setup.mjs.orig	2026-06-26 09:32:07
+++ node_modules/yun-elp/es/src/table/src/table-script-setup.mjs	2026-06-26 09:32:07
@@ -70,9 +70,15 @@
 				...propsPaginationProps
 			};
 		});
-		const paginationChange = (currentPage, pageSize) => {
+		const handleCurrentChange = (currentPage) => {
 			emit("paginationChange", {
 				currentPage,
+				pageSize: paginationProps.value.pageSize
+			});
+		};
+		const handleSizeChange = (pageSize) => {
+			emit("paginationChange", {
+				currentPage: 1,
 				pageSize
 			});
 		};
@@ -113,7 +119,7 @@
 				createElementVNode("span", null, toDisplayString(unref(t)("table.items")), 1)
 			], 2), createVNode(unref(ElPagination), mergeProps(paginationProps.value, {
 				class: unref(ns).e("footer-pagination"),
-				onChange: paginationChange
+				onCurrentChange: handleCurrentChange, onSizeChange: handleSizeChange
 			}), null, 16, ["class"])], 2)])], 2)) : createCommentVNode("", true)], 2)), [[_directive_loading, __props.loading]]);
 		};
 	}
